recipes/export2graphlan/meta.yaml

```diff
@@ -20,5 +20,11 @@ requirements:
       - scipy
       - numpy
 test:
+  requires:
```

Comment:
I finally removed them. I got an error during `docker` test, but it was due to lack of `numpy` in global requirements. So, I thought I had to had also the requirements for tests. But, it works (at least locally) without them